comp/metadata/packagesigning/packagesigningimpl/testdata/debsig/policies/1A2B3C4D5E/dd-c.pol

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!DOCTYPE Policy SYSTEM "https://www.debian.org/debsig/1.0/policy.dtd">
+<!-- older Debian systems (e.g. Debian 7) require policy xmlns to start with http://, not https:// -->
+<Policy xmlns="http://www.debian.org/debsig/1.0/">
+	<Origin Name="Datadog" id="1A2B3C4D5E" Description="Datadog, Inc" />
+
+	<Selection>
+		<Required Type="origin" File="debsig.gpg" id="1A2B3C4D5E" />
+	</Selection>
+
+	<Verification MinOptional="0">
+		<Required Type="origin" File="debsig.gpg" id="1A2B3C4D5E" />
```

Comment:
Unrelated but I'd be curious if this is what our currently-dropped files do since short IDs have a similar uniqueness concern mentioned in earlier comment

Reply:
[Here](https://github.com/DataDog/datadog-signing-keys/blob/master/files/policies/E6266D4AC0962C7D/datadog.pol) is what we deliver currently. I'll take this uniqueness point with @bkabrda, thanks